swap_meet/item.py

```diff
@@ -1,2 +1,49 @@
+import uuid
+
+# WAVE 2
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):  # default id is None
+        """
+        Initializes an Item with a unique ID.
+        If no ID is provided, a random unique ID is generated using uuid.
+
+        Written By Mariya Mokrynska
+        """
+        if id == None: 
+            self.id = uuid.uuid4().int  # unique id
+        else:
+            self.id = id
+        self.condition = condition # assign condition value
+
+    def get_category(self):
+        """
+        Returns the name of the class as a string.
+
+        Written By Mariya Mokrynska
+        """
+        return self.__class__.__name__
```

Comment:
A child class can inherit this method if there is no custom behavior that needs to be overrode in the child class.

When we look at `get_category` in `Item` the parent class and all three child classes, we can see they are exactly the same. 

Since they're exactly the same, the three child classes are unnecessarily overriding the parent class's method. We should remove the redundant method `get_category` in all 3 child classes.